Gemfile

```diff
@@ -1,5 +1,6 @@
 source 'https://rubygems.org'
 
 gem "ProMotion", git: "https://github.com/clearsightstudio/ProMotion.git", branch: "master"
+gem 'motion_print'
```

Comment:
I think the ProMotion gem version it should have `motion_print` already as a dependency